Frontend/src/locales/en.json

```diff
@@ -76,9 +76,21 @@
   },
   "photos":  {
     "myPhotos": "My Photos",
-    "friendsPhotos": "Photos your friends took"
+    "descriptionMyPhotos": "Here you can view all the photos you've uploaded for the Photowalk challenges. Photos your Friends like the best have the most heart icons.",
+    "friendsPhotos": "Photos my friends took",
+    "descriptionFriendsPhotos": "Here you can view all Photos your Friends took on their Photowalks. Do you recognize any of the motifs? Show your friends you like a Photo they took by klicking on the heart icon!"
```

Comment:
Here you can view all photos your friends took on their Photowalks. Do you recognize any of the motifs? Show your friends that you like a photo they took by clicking on the heart icon!